### DISCUSSION TITLE
Is there an explanation of how 'popularity' is determined?

### DISCUSSION THREAD

**Discussion by Moaz31** [johnwmillr/LyricsGenius#232] [Q&A]
This is less of a bug question and more a ask for clarity.

We can sort by 'popularity' but I have been unable to find any documentation or explanation of how 'popularity' is determined. Is it number of streams? Number of hits on the artists lyrics?

Thank you!

**Comment by allerter** [comment]
They're probably sorted by the number of views. You can ask on the [API Support forum on Genius](https://genius.com/api-support) for a definitive answer.

**Reply by Moaz31** [reply]
Thanks. Am i crazy or is there no button to ask a question on that forum? I cant find it for the life of me :D

**Reply by allerter** [reply]
You have to sign in first. Then, you can click on the [Create discussion](https://genius.com/groups/api-support/discussions/new) button to create a post there.
